chain/network/build.rs

```diff
@@ -0,0 +1,49 @@
+use sha2::Digest;
+
+fn content_prefix(content: &str) -> String {
```

Comment:
There's a weak consensus for top-down code: placing entry-points and public API at the top, and helpers and impl details at the bottom:

https://near.zulipchat.com/#narrow/stream/300659-Rust-.F0.9F.A6.80/topic/Code.20style.20question/near/260109589